packages/model/src/common/index.ts

```diff
@@ -5,6 +5,7 @@
  */
 
 export * from "./DataModelPath.js";
+export * from "./Datatype.js";
```

Comment:
Ack, this file is supposed to be deleted, must've been a bad merge.  Use e.g. `uint8.id` instead of `Datatype.uint8`.